src/components/authetication/AuthenticationCard.tsx

```diff
@@ -1,6 +1,8 @@
 import React, { useState } from "react";
 import { Card, Button, Form } from "react-bootstrap";
 import { useRouter } from "next/router";
+import { emailLogin } from "../../utilities/auth";
+import LoginForm2 from "./LoginForm";
```

Comment:
Rename this to something more appropriate. LoginForm is fine, LoginSection should be used below